packages/components/src/post-stats-card/style.scss

```diff
@@ -0,0 +1,110 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "../styles/typography";
+
+$card-padding: 24px;
+$border-radius: 5px; // stylelint-disable-line scales/radii
+
+.post-stats-card {
+	border-color: var(--studio-gray-5);
+	border-radius: $border-radius;
+	display: grid;
+	font-family: $font-sf-pro-text;
+	font-size: $font-body-small;
+	grid-template-columns: minmax(10px, 1.5fr) minmax(0, auto);
+	grid-template-areas:
+		"heading thumbnail"
+		"post thumbnail"
+		"counts thumbnail";
+	margin-bottom: 0;
+	max-width: 100%;
+	min-width: 180px; // Minimum mobile width
+	max-height: 255px;
+	padding: $card-padding;
+}
+
+.post-stats-card__heading {
+	color: var(--studio-gray-100);
+	font-family: $font-sf-pro-display;
+	font-size: $font-size-header-small;
+	font-weight: 500;
+	grid-area: heading;
+	line-height: 1.3;
+	margin-bottom: $card-padding;
+	padding-right: $card-padding;
+}
+
+.post-stats-card__post-title {
+	font-family: $font-recoleta;
+	font-size: $font-size-header;
+	font-weight: 400;
+	line-height: 40px;
```

Comment:
It looks like the line-height is shared between the title and the value, so maybe it could be a variable.